axolotl-web/src/store/store.js

```diff
@@ -246,6 +265,9 @@ export default new Vuex.Store({
         else if (Object.keys(messageData)[0] == "MessageRecieved") {
           this.commit("SET_MESSAGE_RECIEVED", messageData["MessageRecieved"]);
```

Comment:
I had a `Uncaught TypeError: can't access property "push", tmpList is null    SET_MESSAGE_RECIEVED store.js:167` here in my conversation with all messages failing. Not sure how to reproduce exactly.

Reply:
mhm, that's strange, i need to try to reproduce.

Reply:
I didn't have it again at the moment, I'll open another issue if it happens again.